community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf

```diff
@@ -171,21 +171,41 @@ locals {
 
   slurm_gcp_devel_zip        = "slurm-gcp-devel.zip"
   slurm_gcp_devel_zip_bucket = format("%s/%s", local.bucket_dir, local.slurm_gcp_devel_zip)
+  slurm_gcp_devel_zip_files = toset([
+    for f in fileset(local.scripts_dir, "**") : f if !startswith(f, "tests/")
+  ])
+}
+
+data "local_file" "startup_sh" {
```

Comment:
Why does it needs to be in devel ?

Reply:
This is currently [referred locally](https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tpu.py#L216-L219) when creating a TPU node. I'm adding it here for a sake of completeness and improve development. Otherwise you need to rebuild controller image each time you change this script (or do the change manually).

TPUs do not have yet any templates to keep them there :-/

Reply:
With a dedicated script, there are no changes needed here.